src/database/models/extra_functionality_movie.py

```
@@ -0,0 +1,47 @@
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    user_id: Mapped[int] = mapped_column(ForeignKey("users.id", ondelete="CASCADE"), nullable=False)
+    movie_id: Mapped[int] = mapped_column(ForeignKey("movies.id", ondelete="CASCADE"), nullable=False)
+
+    user: Mapped["UserModel"] = relationship("UserModel", back_populates="favorite_movies")
+    movie: Mapped["MovieModel"] = relationship("MovieModel", back_populates="favorited_by")
+
+    __table_args__ = (UniqueConstraint("user_id", "movie_id", name="unique_favorite"),)
+
+
+class MovieRating(Base):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)